### PR TITLE
add sample profile to helm/kustomize Cognito-rds-s3

### PR DESCRIPTION
**Pull request by ryansteakley**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
Keep parity with Terraform deployment. Adds a sample user-profile configured with s3-bucket access to helm/kustomize cognito-rds-s3 deployments

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/e2e/resources/installation_config/cognito-rds-s3.yaml [hunk 1]
```diff
@@ -361,6 +361,15 @@ aws-authservice:
         - key: app
           value: aws-authservice
 
+#user namespace
+user-namespace:
```
Comment (surajkota) [inline tests/e2e/resources/installation_config/cognito-rds-s3.yaml:365]: please add to cognito as well

Comment (ryansteakley) [reply tests/e2e/resources/installation_config/cognito-rds-s3.yaml:365]: added to all cognito

File: tests/e2e/resources/installation_config/cognito-rds-s3-static.yaml [hunk 2]
```diff
@@ -318,6 +318,17 @@ profiles-and-kfam:
         - key: kustomize.component
           value: profiles
 
+
+#user namespace
+user-namespace:
+  installation_options:
+    kustomize:
+      paths: 
+        - ../../awsconfigs/common/user-namespace/overlay
```
Comment (surajkota) [inline tests/e2e/resources/installation_config/cognito-rds-s3-static.yaml:327]: static one should point to upstream not the aws overlay. same for cognito

Can you also update the Cognito guide to create a user with email user@example.com? https://awslabs.github.io/kubeflow-manifests/docs/deployment/cognito/manifest/guide/#60-connect-to-the-central-dashboard

**Review by surajkota — CHANGES_REQUESTED** [review]

File: tests/e2e/resources/installation_config/cognito-rds-s3-static.yaml [hunk 3]
```diff
@@ -318,6 +318,17 @@ profiles-and-kfam:
...
 
+
+#user namespace
+user-namespace:
+  installation_options:
+    kustomize:
+      paths: 
+        - ../../upstream/common/user-namespace/base
+    helm: 
+      paths: ../../charts/common/user-namespace
```
Comment (surajkota) [inline tests/e2e/resources/installation_config/cognito-rds-s3-static.yaml:329]: add if else for the plugin portion of the profile

**Review by surajkota — APPROVED** [review]

**Comment by ryansteakley** [comment]
Drafting doc changes in separate pr

**Pull request merged**